tests/unit/clients/memcache_tests.py

```diff
@@ -10,11 +10,61 @@
 else:
     del pymemcache
 
+from prometheus_client import REGISTRY
 from baseplate.lib.config import ConfigurationError
 from baseplate.clients.memcache import pool_from_config
+from baseplate.clients.memcache import MonitoredMemcacheConnection
 from baseplate.clients.memcache import lib as memcache_lib
 
 
+class PrometheusInstrumentationTests(unittest.TestCase):
+    def test(self):
+        span = mock.Mock()
```

Comment:
It would be great to mock out the span more precisely here if possible. Ideally this test would fail if the span API changed in such a way that the metrics could not be collected.

Reply:
the prom metrics get collected regardless of the span, but each function creates a new span so it needs to be stubbed out. added a comment to that effect